src/pybind/mgr/dashboard_v2/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -42,6 +42,19 @@ <h1 translate>Welcome to ceph!</h1>
                *ngIf="(loginForm.submitted || password.dirty) && password.invalid">Password is required</div>
         </div>
 
+        <!-- Stay signed in -->
+        <div class="form-group">
+          <div class="checkbox checkbox-primary">
+            <input id="stay_signed_in"
```

Comment:
The checkbox is not aligned.
![capture](https://user-images.githubusercontent.com/399326/35560314-fdfe6ab4-05a4-11e8-8c24-7f0b25759f77.PNG)

Reply:
All fixed